definitions.mk

```diff
@@ -1,16 +1,32 @@
 RTEMS_API = 6
 RTEMS_CPU = sparc
 RTEMS_BSP = gr712rc-qual-only
-RTEMS_ROOT = /opt/rtems-6-sparc-gr712rc-smp-3
+RTEMS_ROOT = /opt/rtems-6-sparc-gr712rc-smp-4
 
 SIS_NAME = sis
 SIS_VERSION = 2.29
 
-AR = $(RTEMS_ROOT)/sparc-rtems6/bin/ar
-AS = $(RTEMS_ROOT)/sparc-rtems6/bin/as
+SRC_DIR = src
+BUILD_DIR = build
+TEST_DIR = test
+UNIT_TEST_DIR = unit
+INTEGRATION_TEST_DIR = integration
+MOCK_DIR = mock
+SIS_MODULE_SRC_DIR = sis
+UART_SRC_DIR = Uart
+UTILS_SRC_DIR = Utils
+SYSTEM_CONFIG_SRC_DIR = SystemConfig
+
+G++ = g++
```

Comment:
G++? Are we sure we don't want to use CXX/CPP? And if we are differentiating between native and cross compiler, then maybe use convention:
NATIVE/X86/HOST_CXX=g++
CROSS/SPARC_CXX= $(RTEMS_CPU)-rtems$(RTEMS_API)-g++

Reply:
ack